templates/README.md

```diff
@@ -3,11 +3,8 @@
 ## Setup
 
  ```sh
-$ bundle install
-$ bundle exec rake db:create
-$ bundle exec rake db:migrate
-$ bundle exec rake db:seed
-$ bundle exec rails s
+$ bin/setup
+$ bin/dev
```

Comment:
TIL

Reply:
Rails 7 uses Foreman to run multiple actions, and as such `bin/dev` is now the standard!